### PR TITLE
Use filestream input instead of deprecated log

### LINKED ISSUE #67: [Documentation] Setup guide recommends the deprecated Filebeat `log` input instead of the new `filestream` input
The [ECS Logging Java Setup](https://www.elastic.co/guide/en/ecs-logging/java/current/setup.html) recommends the Filebeat [`log` input](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-log.html) that is deprecated since 7.16 in favor of the [`filestream` input](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-filestream.html)

### PR DESCRIPTION
**Pull request by felixbarny**
Fixes #67

### CODE REVIEW

File: docs/tab-widgets/filebeat.asciidoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 .filebeat.yaml
 ----
 filebeat.inputs:
-- type: log
+- type: filestream
   paths: /path/to/logs.json
   json.keys_under_root: true
```
Comment (cyrille-leclerc) [inline docs/tab-widgets/filebeat.asciidoc]: `filestream` doesn't have the same syntax for the json tricks.

it looks like

```
- type: filestream
  paths: /path/to/logs.json
  parsers:
      - ndjson:
          keys_under_root: true
          overwrite_keys: true
```

See https://github.com/cyrille-leclerc/my-shopping-cart/blob/2cde6dda3974143271fe9e2d929ffa59849fa94b/elastic-filebeat/filebeat.yml

**Comment by ghost** [comment]
## :green_heart: Build Succeeded



> _the below badges are clickable and redirect to their specific view in the CI or DOCS_
[Pipeline View](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-shared%2Fecs-logging-update-specs-mbp%2FPR-69/detail/PR-69/4//pipeline) [Test View](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-shared%2Fecs-logging-update-specs-mbp%2FPR-69/detail/PR-69/4//tests) [Changes](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-shared%2Fecs-logging-update-specs-mbp%2FPR-69/detail/PR-69/4//changes) [Artifacts](https://apm-ci.elastic.co/blue/organizations/jenkins/apm-shared%2Fecs-logging-update-specs-mbp%2FPR-69/detail/PR-69/4//artifacts) [preview](http://ecs-logging_69.docs-preview.app.elstc.co/diff)  [preview](https://ci-stats.elastic.co/app/apm/services/apm-ci/transactions/view?rangeFrom=2022-04-04T04:25:00.842Z&rangeTo=2022-04-04T04:45:00.842Z&transactionName=apm-shared/ecs-logging-update-specs-mbp/PR-{number}&transactionType=job&latencyAggregationType=avg&traceId=5fba7514af8ac57ccf461a8e25575839&transactionId=5b25ae1efba383b9)


<details><summary>Expand to view the summary</summary>
<p>

#### Build stats




* Start Time: 2022-04-04T04:35:00.842+0000


* Duration: 3 min 8 sec





</p>
</details>

**Review by cyrille-leclerc — CHANGES_REQUESTED** [review]
I think we have to document both filebeat <7.16 with the `log` input and filebeat >= 7.16 with the `filestream` input because bumping filebeat may not be easy.

File: docs/tab-widgets/filebeat.asciidoc (hunk 1 above)
Comment (felixbarny) [reply docs/tab-widgets/filebeat.asciidoc]: Thanks, I've added two sections now. I've also added instructions for adding metadata processors. LMKWYT.

**Review by cyrille-leclerc — APPROVED** [review]
MAny thanks @felixbarny

**Pull request merged**